openmc/model/surface_composite.py

```diff
@@ -212,7 +212,7 @@ def __pos__(self):
 
 
 class IsogonalOctagon(CompositeSurface):
-    """Infinite isogonal octagon composite surface
+    r"""Infinite isogonal octagon composite surface
```

Comment:
```suggestion
    """Infinite isogonal octagon composite surface
```

is this a typo

Reply:
Nope, it is not a typo. Sorry I forgot to mention this in the description. That docstring includes latex (`\sqrt`, etc.), so the escape sequences need to be interpreted via a [raw string](https://docs.python.org/3/reference/lexical_analysis.html#:~:text=Both%20string%20and%20bytes%20literals%20may%20optionally%20be%20prefixed%20with%20a%20letter%20%27r%27%20or%20%27R%27%3B%20such%20strings%20are%20called%20raw%20strings%20and%20treat%20backslashes%20as%20literal%20characters), otherwise Python will think `\s` is a special character.

Reply:
ah yes looking at the rest of the unedited doc string the \r makes sense, So the doc string was previously broken, thanks for fixing it up